src/createServer.js

```diff
@@ -1,3 +1,64 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable no-console */
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  return http.createServer((request, response) => {
+    const normalizeUrl = new URL(request.url, `http://${request.headers.host}`);
+    const toCase = normalizeUrl.searchParams.get('toCase');
+    const text = normalizeUrl.pathname.slice(1);
+    const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+    response.setHeader('Content-Type', 'application/json');
+
+    const errors = [];
+
+    if (!text) {
+      errors.push({
+        message:
+          'Text to convert is required. ' +
+          'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!toCase) {
+      errors.push({
+        message:
+          '"toCase" query param is required. ' +
+          'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    } else if (!cases.includes(toCase)) {
+      errors.push({
+        message:
+          'This case is not supported. ' +
+          'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+      });
```

Comment:
for better maintainability instead of using the hardcoded string "SNAKE, KEBAB, CAMEL, PASCAL, UPPER." you should use cases array. This ensures that the list of valid cases is consistent and easier to update